ui/ui-frontend/projects/archive-search/src/app/archive/filing-holding-scheme/leaves-tree/leaves-tree.component.ts

```diff
@@ -254,6 +265,12 @@ export class LeavesTreeComponent implements OnInit, OnChanges, OnDestroy {
       : '';
   }
 
+  colorOrphanNode(node: FilingHoldingSchemeNode) {
+    let cssId = 'filing-holding-scheme-tree-node-selected';
+    if (!this.nestedTreeControlLeaves.isExpanded(node)) cssId = cssId.concat(' selected-node');
```

Comment:
Je préfère avoir des constantes :
```suggestion
    const expanded = this.nestedTreeControlLeaves.isExpanded(node);
    const cssId = `filing-holding-scheme-tree-node-selected${expanded ? '' : ' selected-node'}`;
```